app/auth/signin/page.js

```diff
@@ -2,24 +2,30 @@
 "use client";
 
 import { set } from "mongoose";
-import { signIn } from "next-auth/react";
+import { signIn, signOut } from "next-auth/react";
 import { useSearchParams, useRouter } from "next/navigation";
-import { ChangeEvent, useState } from "react";
+import { useState, useEffect } from "react";
 import styles from "../../../styles/scss/signin.module.scss";
 import Link from "next/link";
 import Navbar from "../../../components/pages/Navbar";
-import Icon from "../../../components/application/widgets/Icon";
+import { useSession } from "next-auth/react";
+import ThirdPartySignInButtons from "../../../components/pages/Signin/ThirdPartySignInButtons";
 
 const SignInPage = () => {
+  const { data: session } = useSession();
+  const [signUpInitiated, setSignUpInitiated] = useState(false);
+
   const [loading, setLoading] = useState(false);
   const [formValues, setFormValues] = useState({
     email: "",
     password: "",
   });
   const [error, setError] = useState("");
+  const [notification, setNotification] = useState(null);
   const searchParams = useSearchParams();
   const callbackUrl = searchParams.get("callbackUrl") || "/";
   const router = useRouter();
+  const [result, setResult] = useState(null); // 初始化 result 状态
 
```

Comment:
The result and setResult are defined but never used.

Reply:
Done